examples/hello_world/comment.md

```diff
@@ -0,0 +1,12 @@
+## Comments
+
+V supports single line comments `//` and mutli-line comments `/* */`. They should be used for documenting the code for letting the other users know how the code works. It can also be used for temporarily commenting the code which has to be used later on.

```

Comment:
```suggestion
V supports single line comments `//` and multi-line comments `/* */`. They should be used for documenting the code for letting the other users know how the code works. It can also be used for temporarily commenting the code which has to be used later on.
```